src/ListSelector.js

```diff
@@ -31,37 +41,77 @@ export function ListSelector(props) {
     function deleteCurrentList(id) {
         props.changeCurrentList(null);
         props.collection.doc(id).delete();
-        console.log("id: ", id);
         props.changeCurrentPage("checklist");
     }
 
 
+    function removePersonFromList(email, newPage) {
+        props.collection.doc(props.currentList).update({
+            sharing: getListField("sharing").filter(item => item !== email),
+        });
+        props.changeCurrentPage(newPage);
+        if(newPage === "checklist") {
+            props.changeCurrentList(null);
+        }
+    }
+
+    function getListField(field) {
+        return (listsOwned.concat(listsShared)).filter(list => list.id === props.currentList)[0][field];
+    }
+
+    function shareList(email) {
+        props.collection.doc(props.currentList).update({
+            sharing: [...getListField("sharing"), email],
+        });
+    }
+
     return (
         <div>
-            {!loading && props.currentPage !== "newList" && props.currentList !== null && lists.filter(list => list.id === props.currentList)[0].title}
+            {!loadingShared && !loadingOwned && props.currentPage !== "newList" && props.currentList !== null &&
+            <>
+                <h1>{getListField("title")}
+                {listsShared.filter(item => item.id === props.currentList).length > 0 && " (shared)"}</h1>
+            </>}
             {props.currentPage === "checklist" && <div id="sort-by">Select a list:
                 <select name="listSelector" aria-label="dropdown to select list" id="listSelectorDropdown"
                         onChange={(e) => props.changeCurrentList(e.target.value)}>
                     {props.currentList === null && <option disabled selected value>Select a list...</option>}
-                    {lists.map(item => <option value={item.id}
+                    {listsOwned.map(item => <option value={item.id}
                                                selected={item.id === props.currentList}>{item.title}</option>)}
+                    {listsShared.map(item => <option value={item.id}
+                                                    selected={item.id === props.currentList}>{item.title + " (shared)"}</option>)}
                 </select>
-                <button id="addButton" aria-label="Add new list" onClick={() => props.changeCurrentPage("newList")}>
+                {props.currentList !== null && <span>
```

Comment:
This code only shows the add new list button if there is already a current list.

New users don't have a current list and so can't ever create any lists!